test/Sdk.Generator.Tests/RoslynTestUtils.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis;
+using Xunit;
+
+namespace Microsoft.Azure.Functions.SdkGeneratorTests
+{
+    /// <summary>
+    /// This is a trimmed down version of the TestUtils from the dotnet runtime repo.
```

Comment:
This is a trimmed version of the test util class from dotnet runtime repo. We shall augment with more methods from the origin source as needed.